libs/platform/experience/src/services/experience.providers.ts

```diff
@@ -55,11 +65,34 @@ export const layoutProviders: Provider[] = [
   },
 ];
 
-export const experienceProviders: Provider[] = [
+export const adapterProviders: Provider[] = [
```

Comment:
if it's about "Contentul as EB backend" providers, I'd consider creating feature for it. As 'adapterProviders' sound too generic IMO.